pkg/controller/infinispan/util/kubernetes.go

```diff
@@ -199,6 +199,26 @@ func (k Kubernetes) PublicIP() string {
 	return u.Hostname()
 }
 
+// GetNodesHost return the addresses of the k8s nodes
+func (k Kubernetes) GetNodesHost() []string {
```

Comment:
@rigazilla why we need a slice if we always access the first item? Or it just for future use?

Reply:
@dmvolod yeah, maybe it can be used somewhere else